app/Livewire/Upgrade.php

```
@@ -41,4 +51,71 @@ public function upgrade()
+            ];
+        }
+
+        $stepInt = (int) $step;
+        $status = $stepInt >= 6 ? 'complete' : 'in_progress';
+
+        return [
+            'status' => $status,
+            'step' => $stepInt,
+            'message' => $message,
+        ];
+    }
```

PHPMD (2.15.0)

The method getUpgradeStatus() has a Cyclomatic Complexity of 10. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)